README.md

```diff
@@ -37,8 +62,11 @@ The Vechain Improvement Proposals (VIPs) repository can be found [here](https://
 
 To chat with other community members you can join:
 
-- [Telegram](https://t.me/VechainDevCommunity)
-- [Discord](https://discord.gg/vechain)
+<p>
+    <a href="https://discord.gg/vechain"><img src="https://img.shields.io/badge/Discord-5865F2?style=for-the-badge&logo=discord&logoColor=white" /></a>
+    <a href="https://t.me/VechainDevCommunity"><img src="https://img.shields.io/badge/Telegram-2CA5E0?style=for-the-badge&logo=telegram&logoColor=white" /></a>
```

Comment:
We had migrated to the new telegram, which contains a new dev chat channel https://t.me/vechainandfriends, do we need to promote this?